compatibility_server/Dockerfile

```diff
@@ -34,4 +34,4 @@ EXPOSE 8888
 # created at most in the vm.
 CMD cd compatibility_checker && \
   gunicorn -b 0.0.0.0:8888 -w 10 --worker-class gevent --max-requests 20 --max-requests-jitter 10 --timeout 300 \
-           compatibility_checker_server:app
+           compatibility_checker_server:app --export_metrics
```

Comment:
This is not working... We cannot pass the application args like this, see https://stackoverflow.com/questions/8495367/using-additional-command-line-arguments-with-gunicorn